src/test/functional/features/final-order.feature

```diff
@@ -88,4 +88,9 @@ Feature: Final order
     When I login with applicant "1"
     Then the page should include "Your husband has applied for a ‘final order’."
     And the page should include "A judge will review the application"
-    And the page should include "You will then receive an email telling you what they decide."
\ No newline at end of file
+    And the page should include "You will then receive an email telling you what they decide."
+
+    Given I set the case state to "FinalOrderComplete"
+    When I go to "/respondent/hub-page?lng=cy"
```

Comment:
remove the query

Reply:
@velimd This ticket is for Welsh only and the English version of this ticket is not done yet as it is dependent on another ticket. Without the query it will fail as the conditions do not meet?